packages/sources/coinpaprika/src/endpoint/crypto-single.ts

```diff
@@ -102,7 +118,13 @@ export const execute: ExecuteWithConfig<Config> = async (request, context, confi
   }
 
   const response = await Requester.request<ResponseSchema>(options)
-  const result = Requester.validateResultNumber(response.data, resultPath)
+  const coinInfo = response.data as unknown
```

Comment:
What's happening here? Is the API response sometimes an array and sometimes not?

Reply:
I got this figured out and added e2e test coverage.